packages/rum-core/src/domain/action/interactionSelectorCache.ts

```diff
@@ -0,0 +1,21 @@
+import { elapsed, ONE_SECOND, relativeNow } from '@datadog/browser-core'
+import type { RelativeTime } from '@datadog/browser-core'
+
+// Maximum duration for click actions
+export const CLICK_ACTION_MAX_DURATION = 10 * ONE_SECOND
+export const interactionSelectorCache = new Map<RelativeTime, string>()
+
+export const getInteractionSelector = (relativeTimestamp: RelativeTime) => {
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏For consistency sake, you could use regular function (not arrow) :)